packages/react-router-navigation/src/TypeDefinitions.js

```diff
@@ -45,6 +45,7 @@ export type NavigationProps = NavBarProps & {
   ) => NavigationTransitionSpec,
   onTransitionStart?: (...args: Array<mixed>) => void,
   onTransitionEnd?: (...args: Array<mixed>) => void,
+  mode?: string,
```

Comment:
i think the following would be better:
```js
mode?: 'card' | 'modal'
```

Reply:
My bad I don't use Flow... Indeed it works now, nice!